lms/envs/common.py

```diff
@@ -1578,16 +1598,40 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 ################################# WIKI ###################################
 from lms.djangoapps.course_wiki import settings as course_wiki_settings  # pylint: disable=wrong-import-position
 
+# .. toggle_name: WIKI_ACCOUNT_HANDLING
+# .. toggle_implementation: Django setting
+# .. toggle_default: True
+# .. toggle_description: This setting allows the access to Sign up, login and logout views.
+# .. toggle_warnings: Turn to False only if you have your own account handling
```

Comment:
I would just drop this.  As noted above, I don't even know what would happen if this were set to `True`.